### PR TITLE
Rasterize improvements

### PR DESCRIPTION
**Pull request by nb-pdragoi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Improvements to the main rasterization integration which gives the ability to configure the filename. This is useful because running multiple rasterization commands in the same context will cause the resulting files to overlap and be hard to be used for even further automation.

## Minimum version of Cortex XSOAR
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 1]
```diff
@@ -296,10 +296,11 @@ def rasterize_command():
     r_type = demisto.args().get('type', 'png')
     wait_time = int(demisto.args().get('wait_time', 0))
     page_load = int(demisto.args().get('max_page_load_time', DEFAULT_PAGE_LOAD_TIME))
+    filename = demisto.args().get('filename', 'url')
```
Comment (yaakovpraisler) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:299]: Please change the argument name `filename` to `file_name` in all this file.
```suggestion
    file_name = demisto.args().get('file_name', 'url')
```

**Review by yaakovpraisler — COMMENTED** [review]
Looking good! Please see my comments - 

Also, please if you can add unit tests for your changes.

**Comment by yaakovpraisler** [comment]
@pdragoi - Please sign in to the CLA as you can see [above](https://github.com/demisto/content/pull/14124#issuecomment-896129429), so I'll approve and merge this PR.

**Review by yaakovpraisler — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
HI @pdragoi We would like to conclude the efforts here. Please address @yaakovpraisler last comments otherwise if we'll have no response we'll have to close this PR.

**Comment by nb-pdragoi** [comment]
This work of authorship being contributed to the XSOAR content pursuant to the PA CLA is not intended as a contribution to any other project except to the extent NB provides it separately by way of an additional Contribution.

**Comment by ShahafBenYakir** [comment]
@nb-pdragoi As discussed on private, all content in Demisto/content repo is subject to the [MIT license](https://github.com/demisto/content/blob/master/LICENSE) and it will be the only definition for the usage of this content. Please revert to us if you want this PR to be merged or not.

**Comment by nb-pdragoi** [comment]
We can go forward with the merge.

**Pull request merged**